app/config/core.php

```diff
@@ -24,4 +24,8 @@
 define('MAX_TITLE_CHARACTERS', 30);
 define('MIN_PASSWORD_CHARACTERS', 8);
 define('MAX_PASSWORD_CHARACTERS', 20);
```

Comment:
It's been suggested to me that if the constant variable is being by many classes, then it's okay to declare to core.php. But if not, better to declare it to the class where it is concerned po :bow: